.github/workflows/main.yml

```diff
@@ -10,6 +10,7 @@ permissions:
 
 jobs:
   CD:
+    needs: ["run-linters"]
```

Comment:
I think this may not reference correctly but we can try it and see.

Reply:
@suzil, it is not the final i am testing some things

Reply:
ok @suzil